### DISCUSSION TITLE
[Feature Request] <Title>

### DISCUSSION THREAD

**Discussion by Ea1154** [zkSync-Community-Hub/zksync-developers#360] [Feature requests]
### Network

zkSync Era

### Provide a brief description of the feature you would like to see implemented.

<LINK_REDACTED>
"Would you like to see how much gas you paid on zkSync compared to other platforms after completing a transaction?

### Who does this feature request benefit the most (users, developers, protocol, etc.)?

user , developers.

**Comment by kahkaha54** [comment]
It would be nice

**Reply by kahkaha54** [reply]
It would be nice

**Comment by nooma-42** [comment]
I suspect this is a scam link @MexicanAce

**Comment by jablonskibrittany** [comment]
Implementing a feature to display gas fees paid on zkSync compared to other platforms after completing a transaction would benefit both users and developers. It provides users with valuable insight into their transaction costs and allows developers to optimize their applications for cost efficiency. This feature enhances transparency and usability on zkSync Era, benefiting the entire ecosystem.

**Comment by berronesjosh** [comment]
Implementing a gas cost comparison feature on zkSync Era would benefit both users and developers by providing valuable insights into transaction costs and enhancing transparency on the platform.

**Comment by renemilnev** [comment]
Implementing a gas comparison feature on zkSync Era would benefit both users and developers by providing valuable insights into transaction costs and efficiency across different platforms.

**Comment by queskim** [comment]
Sure, we can implement a feature that allows users to compare gas fees paid on zkSync with other platforms after completing a transaction. This will benefit both users and developers.
